src/main/java/seedu/tr4cker/logic/commands/CountdownCommand.java

```diff
@@ -74,6 +86,7 @@ public CountdownCommand() {
         this.eventDate = null;
         this.index = null;
         this.isDeleteCountdown = false;
+        this.numDays = -1;
```

Comment:
Perhaps you can wrap -1 as a variable instead of using it as a magic number in the other methods as well?